### PR TITLE
Refactoring for my bro !

### PR DESCRIPTION
**Pull request by EmirhanAksoy**
Hey dude , I have just check your repo and made some changes , please check it ! 

Have a nice day 👍

### CODE REVIEW

**Comment by nooranshah** [comment]
thanks bro jan

File: OOPAssignment/Base/Surface.cs [hunk 1]
```diff
@@ -9,7 +9,8 @@ public class Surface : ISurface, ICollidableSurface, IBase.IObserver<CarInfo>
     {
         public long Height { get; set; }
         public long Width { get; set; }
-        private List<CarInfo> ObservableCars = new List<CarInfo>();
+
```
Comment (nooranshah) [inline OOPAssignment/Base/Surface.cs:12]: is it depends on .core version 6 👍

**Comment by nooranshah** [comment]
thanks again my wonderful bro jan 👍

File: OOPAssignment/Base/Surface.cs (hunk 1 above)
Comment (EmirhanAksoy) [reply OOPAssignment/Base/Surface.cs:12]: Nope, **new()** is enough in .NET5

**Pull request merged**